ciao-cert/README.md

```diff
@@ -51,63 +51,66 @@ Usage of ciao-cert:
 
 ## Example
 
-On our example cluster the scheduler is running on ciao-ctl.intel.com.
-ciao.ctl.intel.com is a multi homed machine connected to the cluster
-control plane through 192.168.1.118
+On our example cluster, the scheduler is running on ciao-ctl.intel.com.
+The ciao-ctl.intel.com host is a multi homed machine connected to the
+cluster control plane through 192.168.1.118.
 
-We need to generate 5 private keys for the Scheduler, Controller, Networking Agent,
-CNCI Agent and the Compute Node Agents. We also need to generate the CA certificate.
+At a minimum, need to generate 5 private keys for the Scheduler,
+Controller, Networking Agent, CNCI Agent and the Compute Node Agents. We
+also need to generate the CA certificate.
 
-* Scheduler private key and CA certificate
+Depending on your security tolerances, you may choose one of two deployment
+styles for SSNTP client certificates:
+* a generic private key for each SSNTP client type: ```-host=localhost``` in the command examples below.  Each Compute Node agent machine would have a copy of the one compute node agent key with 'localhost' validity.  This may be easiest for a fleet of ephemeral test systems.
```

Comment:
@tpepper the CNCI cert is not an issue. The scheduler is the only one that has to have a FQDN cert that resolves. As the CNCI agent and in fact even the launchers use to connect back to the scheduler.

So localhost should not be used for the scheduler cert.